docs/standard/library-guidance/dependencies-net.md

```diff
@@ -0,0 +1,169 @@
+---
+ai-usage: ai-assisted
+---
+# Guidance for .NET library dependency versions
+
+When building libraries that target multiple .NET versions, choosing dependency versions has implications for compatibility, servicing, and ecosystem health. This document outlines the main strategies, their tradeoffs, and recommendations.
+
+---
+
+## **Overview**
+
+Library authors face challenges when deciding which version of a .NET dependency to reference. Newer versions have more API and features, but may require a local redistribution increasing servicing responsibilities of the library and size of the application.  The decision impacts:
+
+- **Friction of updates** on older runtimes
```

Comment:
This list of decision impacts is only focusing on the negatives. It doesn't touch on the benefits of utilizing newer features and APIs, bug fixes, performance improvements, etc., nor on what folks need to do to work around the lack of such things, like polyfills, which can be both hard to get right (if possible to get 100% right) and require substantial amounts of code.